lib/fizzBuzz.js

```diff
@@ -3,7 +3,26 @@
 Object.defineProperty(exports, "__esModule", {
```

Comment:
Might want to put the `lib` directory in your `.gitignore` so those files don't clutter up your repo.